lib/vdsm/virt/thinp.py

```diff
@@ -337,15 +387,35 @@ def _should_extend_volume(self, drive, volumeID, block_info):
         # size. It should be noted that allocation cannot be directly
         # compared with the volume physical size as it includes also the
         # clusters not written yet (pending).
-        if block_info.allocation > nextPhysSize:
+        if drive.block_info.allocation > nextPhysSize:
             self._vm.pause(pauseCode='EOTHER')
             raise ImprobableAllocationError(
                 "Improbable allocation, pausing the VM to avoid corruption: "
-                f"{block_info}")
+                f"{drive.block_info}")
 
-        free_space = block_info.physical - block_info.allocation
+        free_space = drive.block_info.physical - drive.block_info.allocation
         return free_space < drive.watermarkLimit
 
+    def _extend_in_progress(self, drive):
```

Comment:
This means the extend *may* be still running but is not necessarily still running, right? If so then the naming is confusing, something like ``_recently_started_extend`` would be more accurate.

Reply:
Yes, this tells if extend may be in progress, I'll rename.